policyengine_us/variables/gov/states/mi/tax/income/deductions/standard/tier_two/mi_standard_deduction_tier_two.py

```diff
@@ -0,0 +1,66 @@
+from policyengine_us.model_api import *
+
+
+class mi_standard_deduction_tier_two(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan tier two standard deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "http://legislature.mi.gov/doc.aspx?mcl-206-30",  # (b)&(c)
+        "https://www.michigan.gov/taxes/iit/retirement-and-pension-benefits/michigan-standard-deduction",
+        "https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/BOOK_MI-1040.pdf#page=15",
+    )
+    defined_for = StateCode.MI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mi.tax.income.deductions.standard.tier_two
+        # Core deduction based on filing status.
+        filing_status = tax_unit("filing_status", period)
+        # HAVE TO CHECK THIS
+        sd2_age_eligibility = tax_unit(
+            "mi_standard_deduction_tier_two_eligible", period
+        )
+        age_older = tax_unit("greater_age_head_spouse", period)
+        age_threshold = age_older >= p.min_age
+        # (9), (b)
+        # If the person has not reached the age of 67 (which is mathematically impossible) and is born betwee 1946 and 1952
```

Comment:
need a bool check for lines 23C and 23G - to receive the additional 15,000 or 30,000